### PR TITLE
use dynamic image start-trial-cta in css

### PR DESCRIPTION
**Pull request by AucT**
## Summary by CodeRabbit

- **New Features**
	- Introduced a new CSS file for dynamic background images on specific elements, enhancing visual appeal.
	- Added a new background image to improve the styling of specific layout elements.
  
- **Bug Fixes**
	- Removed unnecessary background image from a layout element, streamlining the design.

- **Refactor**
	- Updated the 404 error page layout to include dynamic CSS for improved styling flexibility.
	- Enhanced CSS loading mechanism to better handle dynamic styles based on context.

### CODE REVIEW

File: themes/beaver/assets/css/3082-layout.css [hunk 1]
```diff
@@ -940,7 +940,7 @@ img.mfp-img {
 }
 
 .fl-node-ec25np07hf63 > .fl-row-content-wrap {
-
+  background-image: url(/wp-content/uploads/2022/10/white-border-background.webp);
```
Comment (pftg) [inline themes/beaver/assets/css/3082-layout.css]: Please rebase I deleted it

Comment (AucT) [reply themes/beaver/assets/css/3082-layout.css]: no need, I used it only for 404.css, 590-layout.css

Comment (pftg) [reply themes/beaver/assets/css/3082-layout.css]: I deleted those files. You can just delete this line

Comment (AucT) [reply themes/beaver/assets/css/3082-layout.css]: forced push without changes to this file
now should be purrfect

File: themes/beaver/layouts/page/index.html [hunk 2]
```diff
@@ -24,9 +24,10 @@
 
    document.head.appendChild(link);
 }
-
+{{- $dynamicCSS := resources.Get "css/dynamic-404-590.css" | resources.ExecuteAsTemplate "css/dynamic.css" . }}
```
Comment (pftg) [inline themes/beaver/layouts/page/index.html:27]: Are we using start cta only on those 2 pages? Other pages do not have such block?

Comment (AucT) [reply themes/beaver/layouts/page/index.html:27]: only 2 pages

**Pull request merged**